src/App.jsx

```diff
@@ -1,44 +1,31 @@
-import React from 'react';
+import { useState } from 'react';
+
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs/Tabs';
```

Comment:
You have the following code:
```javascript
// ./components/Tabs/index.js
export * from './Tabs';
```

Because of this, you don’t need to import it like this:
```javascript
// App.jsx
import { Tabs } from './components/Tabs/Tabs';
```

Instead, you can simplify the import to:
```suggestion
import { Tabs } from './components/Tabs';
```

Using an `index.js` file lets you group and re-export files from a folder. This makes your imports shorter and easier to manage as your project grows.